### ISSUE TITLE
trademark logo scan

### ISSUE THREAD

**Issue by dirkmueller** [openSUSE/cavil#54]
is cavil able to scan for trademarked logos and could flag them for manual review? trademark usage is difficult as fair-use could apply, however I think currently we're not highlighting those at all?

**Comment by kraih** [comment]
You mean like scanning all images included in a package with an ML model to identify trademark signs?

**Comment by kraih** [comment]
We can probably do this pretty easily now with a vision model like [Pixtral](https://huggingface.co/mistralai/Pixtral-12B-2409), it would just take hardware resources we don't have at the moment.

**Comment by dirkmueller** [comment]
It doesn't have to be such a large model. I've been running that on a CPU with not much issues

**Comment by kraih** [comment]
Fair enough, would be interesting to test if using CPU for inference could be cost effective. Which smaller vision model were you referring to? Only one with the right license i'm aware of would be Phi-3-vision, which might be a bit too small.
